Environment-Definitions/TerraformTemplates/AzureDataLabs_AzureSynapse/infra/environment.yml

```diff
@@ -0,0 +1,45 @@
+name: AzureDataLabs_AzureSynapse
+summary: Terraform example for Azure Data Labs - Data Analytics with Azure Synapse
+runner: Terraform
+templatePath: main.tf
+parameters: 
+- id: "prefix"
+  name: "Prefix"
+  type: "string"
+  required: true
+- id: "postfix"
+  name: "Postfix"
+  type: "string"
+  required: true
+- id: "location"
```

Comment:
We should be passing [this variable](https://github.com/Azure/deployment-environments/blob/custom-runner-private-preview/Runner-Images/Terraform/scripts/deploy.sh#L27C1-L32C57) in already. Do we want to just use that?